platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/attribution/Attribution.java

```diff
@@ -2,13 +2,21 @@
 
 import android.support.annotation.Nullable;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class Attribution {
 
   private static final String OPENSTREETMAP = "OpenStreetMap";
   private static final String OPENSTREETMAP_ABBR = "OSM";
   static final String TELEMETRY = "Telemetry Settings";
 
-  static final String IMPROVE_MAP_URL = "https://www.mapbox.com/feedback/";
+  static final List<String> IMPROVE_MAP_URL = new ArrayList<String>(){{
+    // Using a List makes URL backwards compatible
+    add("https://www.mapbox.com/feedback/");
```

Comment:
Could you move those to the `static` block?

Reply:
capturing from some searches it does introduce some additional class file. Will change.

Reply:
Correct, it introduces an unnecessary subclass of the referenced class instead of a simple instance.